bin/check_links

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/perl
+use v5.36;
```

Comment:
This currently causes `xt/author/minimum-version.t` to fail when doing a `dzil test`. Since this seems to be meant as a development tool, excluding it from the built dist might be the easy option.

Reply:
As a development tool if this script is included, it should be put somewhere other than bin/ entirely, IMO.

Reply:
Location isn't important. Move it wherever you like.